sdk/kas_client.go

```diff
@@ -72,9 +76,38 @@ func (k *KASClient) makeRewrapRequest(ctx context.Context, requests []*kas.Unsig
 		return nil, fmt.Errorf("error making rewrap request: %w", err)
 	}
 
+	upgradeRewrapResponseV1(response, requests)
```

Comment:
if we convert v1 responses to v2, is that still compatible with clients expecting v1 responses?

Reply:
They will ignore unknown fields.